keps/sig-instrumentation/0034-distributed-tracing-kep.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Leveraging Distributed Tracing to Understand Kubernetes Object Lifecycles
+authors:
+  - "@Monkeyanator"
+editors:
+  - "@dashpole"
+owning-sig: sig-instrumentation
+reviewers:
+  - "@Random-Liu"
+  - "@bogdandrutu"
+approvers:
+  - "@brancz"
```

Comment:
Since @piosz is also a chair of the SIG should @piosz be listed as an approver?

Reply:
I added him, although I believe only one approver is required.